tests/gateway/test_client.py

```diff
@@ -183,6 +183,7 @@ def test_query_individual_route(gateway, monkeypatch):
         "model": {"name": "text-davinci-003", "provider": "openai"},
         "name": "completions",
         "route_type": "llm/v1/completions",
+        "route_url": None,
```

Comment:
Unrelated to this PR, but temporary fix for failing tests until we set route URL property correctly for GetRoute / SearchRoutes responses in OSS Server

Reply:
I filed https://github.com/mlflow/mlflow/pull/8922